From 192ff2030173bc3b433b0ece9d1fe9e25d43b2c1 Mon Sep 17 00:00:00 2001
From: Kacper Kornet <draenog@pld-linux.org>
Date: Tue, 21 Jun 2011 14:34:52 +0200
Subject: [PATCH 2/6] Remove broken links

When some hook files were removed it resulted in broken links to them.
This commit fixes it.
---
 src/gl-install |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/gl-install b/src/gl-install
index de7aefd..2f73869 100755
--- a/src/gl-install
+++ b/src/gl-install
@@ -97,6 +97,7 @@ for my $repo (`find . -type d -name "*.git" -prune`) {
     # in case of package install, GL_ADMINDIR is no longer the top cop;
     # override with the package hooks
     ln_sf("$GL_PACKAGE_HOOKS/common", "*", "$repo/hooks") if $GL_PACKAGE_HOOKS;
+    foreach(glob "$repo/hooks/*") { -l && ! -e && unlink};
     chmod 0755, "$repo/hooks/update";
 }
 
-- 
1.7.8

